pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1822,21 +1730,23 @@ func (s *AWSCloud) updateInstanceSecurityGroupsForLoadBalancer(lb *elb.LoadBalan
 
 		permissions := []*ec2.IpPermission{permission}
 
-		if add {
-			changed, err := s.ensureSecurityGroupIngress(instanceSecurityGroupId, permissions)
+		if instanceSecurityGroupId != nil && groupId == instanceSecurityGroupId {
+			glog.V(2).Infof("Adding rule for traffic from the load balancer (%s) to instances (%s)", loadBalancerSecurityGroupId, groupId)
```

Comment:
Does this log every time we go through the load-balancer reconciliation loop?  Previously we were pretty careful to log only when we were actually making a change.